pkg/security/podsecuritypolicy/provider.go

```diff
@@ -310,6 +316,10 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		}
 	}
 
+	if !s.psp.Spec.AllowPrivilegeEscalation && sc.AllowPrivilegeEscalation != nil && *sc.AllowPrivilegeEscalation {
```

Comment:
so `psp.AllowPrivilegeEscalation` is just "am i allowed to set securityContext.AllowPrivilegeEscalation" so I think it might be a naming problem